wemix/admin.go

```diff
@@ -1172,14 +1163,21 @@ func (ma *wemixAdmin) calculateRewards(num, blockReward, fees *big.Int, addBalan
 		return
 	}
 
-	// determine coinbase
-	if len(rp.members) > 0 {
-		mix := int(num.Int64()/ma.blocksPer) % len(rp.members)
-		coinbase = &common.Address{}
-		coinbase.SetBytes(rp.members[mix].Reward.Bytes())
+	var blockReward *big.Int
+	if config.IsBrioche(num) {
+		blockReward = config.Brioche.GetBriocheBlockReward(defaultBriocheBlockReward, num)
+	} else {
+		// if the wemix chain is not on brioche hard fork, use the `rewardAmount` from gov contract
+		blockReward = big.NewInt(0).Set(rp.rewardAmount)
```

Comment:
common code

```suggestion
		blockReward = new(big.Int).Set(rp.rewardAmount)
```

Reply:
In Ethereum v1.14.0, `big.NewInt(0).XXX` is rarely used.
For future update, I also suggest `new(big.Int)` than `big.NewInt(0)`
```
eth/filters/filter_test.go
169:				addr:      {Balance: big.NewInt(0).Mul(big.NewInt(100), big.NewInt(params.Ether))},

core/chain_makers_test.go
44:		funds   = big.NewInt(0).Mul(big.NewInt(1337), big.NewInt(params.Ether))
```

Reply:
The other places where `big.NewInt(0).XXX` is used for this PR.
* params/config.go: GetBriocheBlockReward()
* params/config_test.go: TestGetBriocheBlockReward()
* wemix/rewards_test.go: TestDistributeRewards()
* wemix/admin.go: `magic` in var block